src/app/app-common/PageSource/PlainTextFilePageSource.cpp

```diff
@@ -35,7 +35,10 @@ PlainTextFilePageSource::PlainTextFilePageSource(
   const DXResources& dxr,
   KneeboardState* kbs)
   : PageSourceWithDelegates(dxr, kbs),
-    mPageSource(std::make_shared<PlainTextPageSource>(dxr, _("[empty file]"))) {
+    mPageSource(std::make_shared<PlainTextPageSource>(
+      dxr,
+      kbs->GetTextSettings().mFontSize,
```

Comment:
Probably best to just pass in KneeboardState/kbs directly:
- this is consistent with other sources
- the type will then indicate an incorrect parameter; with basic types especially, if's plausible another parameter of the same type may be added. It's nice having the compiler able to detect things
- more extensible for the future:
  - for things other than font size
  - even if font size is always the only thing, if you have the kneeboard state, you have the TextSettings object, which also means you have access to any `Event<...>` on either